src/PrestaShopBundle/Bridge/Smarty/HeaderConfigurator.php

```diff
@@ -148,8 +139,6 @@ public function __construct(
     public function configure(ControllerConfiguration $controllerConfiguration): void
     {
         $controllerConfiguration->templateVars['table'] = $controllerConfiguration->tableName;
-        // @todo: hardcoded route. Should be replaced with some value from configuration
-        $controllerConfiguration->templateVars['current'] = $this->router->generate('admin_features_index');
```

Comment:
So this is now handled via the `buildListConfiguration` parameter right?

Reply:
yes it seems $currentIndex is reused everywhere, so it should be good.